crates/ironrdp-server/src/helper.rs

```diff
@@ -21,13 +21,18 @@ impl TlsIdentityCtx {
     ///
     /// The file format can be either PEM (if the file extension ends with .pem) or DER.
     pub fn init_from_paths(cert_path: &Path, key_path: &Path) -> anyhow::Result<Self> {
-        let certs: Vec<_> = if cert_path.extension().map_or(false, |ext| ext == "pem") {
+        let certs = if cert_path.extension().map_or(false, |ext| ext == "pem") {
             CertificateDer::pem_file_iter(cert_path)
+                .context(format!("reading server cert `{cert_path:?}`"))?
```

Comment:
suggestion: It’s recommended to use `with_context` to make the formatting lazy.

```suggestion
                .with_context(|| format!("reading server cert `{cert_path:?}`"))?
```

Reply:
done, thanks